lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+class Takeaway
+  attr_reader :menu, :orders
+
+  def initialize
+    @menu = { rice: 3, chicken: 2, beans: 3, yam: 2, fish: 4, noodles: 2 }
+    @orders = []
+    @sum = 0
+  end
+
+  def add_orders(choice)
```

Comment:
also maybe a fail method if the choice is not in your menu then it will raise an error before pushing it to orders?